### PR TITLE
Added extra test cases for Bonfire: Chunkey Monkey

### LINKED ISSUE #5386: Bonfire Chunky Monkey - Needs more test cases
I was able to pass the tests with the below code, however, I should not have been able to. Instead of Math.round I should have used Math.ceil

The test cases passed even with my incorrect solution because arr.length/size always rounded to the next higher integer with the given test cases.

Challenge [Bonfire: Chunky Monkey](http://www.freecodecamp.com/challenges/bonfire-chunky-monkey#?solution=function%20chunk(arr%2C%20size)%20%7B%0A%20%20%2F%2F%20Break%20it%20up.%0A%20%20%2F%2Fvar%20sliced%20%3D%20arr.slice(0%2C%20size)%3B%0A%20%20%2F%2Fvar%20multiDimensional%20%3D%20%5B%5D%3B%0A%20%20%2F%2FmultiDimensional.push(sliced)%3B%0A%20%20%2F%2Fvar%20secondSliced%20%3D%20arr.slice(size%2C%20size%20%2B%20size)%3B%0A%20%20%2F%2FmultiDimensional.push(secondSliced)%3B%0A%20%20var%20x%20%3D%200%3B%0A%20%20var%20y%20%3D%20size%3B%0A%20%20var%20multiDimensional%20%3D%20%5B%5D%3B%0A%20%20var%20i%20%3D%201%3B%0A%20%20while%20(i%20%3C%3D%20Math.ceil(arr.length%2Fsize))%20%7B%0A%20%20%20%20multiDimensional.push(arr.slice(x%2C%20y))%3B%0A%20%20%20%20x%20%3D%20x%20%2B%20size%3B%0A%20%20%20%20y%20%3D%20y%20%2B%20size%3B%0A%20%20%20%20i%20%2B%2B%3B%0A%20%20%7D%0A%20%20%0A%20%20return%20multiDimensional%3B%0A%20%20%0A%7D%0A%0Achunk(%5B0%2C%201%2C%202%2C%203%2C%204%2C%205%5D%2C%205)%3B%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.106 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```javascript
function chunk(arr, size) {
  // Break it up.
  //var sliced = arr.slice(0, size);
  //var multiDimensional = [];
  //multiDimensional.push(sliced);
  //var secondSliced = arr.slice(size, size + size);
  //multiDimensional.push(secondSliced);
  var x = 0;
  var y = size;
  var multiDimensional = [];
  var i = 1;
  while (i <= Math.round(arr.length/size)) {
    multiDimensional.push(arr.slice(x, y));
    x = x + size;
    y = y + size;
    i ++;
  }
  
  return multiDimensional;
  
}

chunk([0, 1, 2, 3, 4, 5], 5);

```

### PR DESCRIPTION
**Pull request by charliemantle**
closes #5386

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/basic-bonfires.json [hunk 1]
```diff
@@ -546,6 +546,8 @@
         "assert.deepEqual(chunk([0, 1, 2, 3, 4, 5], 3), [[0, 1, 2], [3, 4, 5]], 'message: <code>chunk([0, 1, 2, 3, 4, 5], 3)</code> should return <code>[[0, 1, 2], [3, 4, 5]]</code>.');",
         "assert.deepEqual(chunk([0, 1, 2, 3, 4, 5], 2), [[0, 1], [2, 3], [4, 5]], 'message: <code>chunk([0, 1, 2, 3, 4, 5], 2)</code> should return <code>[[0, 1], [2, 3], [4, 5]]</code>.');",
         "assert.deepEqual(chunk([0, 1, 2, 3, 4, 5], 4), [[0, 1, 2, 3], [4, 5]], 'message: <code>chunk([0, 1, 2, 3, 4, 5], 4)</code> should return <code>[[0, 1, 2, 3], [4, 5]]</code>.');"
+        "assert.deepEqual(chunk([0, 1, 2, 3, 4, 5, 6], 3), [[0, 1, 2], [3, 4, 5], [6]], 'message: <code>chunk([0, 1, 2, 3, 4, 5, 6], 3)</code> should return <code>[[0, 1, 2], [3, 4, 5], [6]]</code>.');"
+        "assert.deepEqual(chunk([0, 1, 2, 3, 4, 5, 6, 7, 8], 4), [[0, 1, 2, 3], [4, 5, 6, 7], [8]], 'message: <code>chunk([0, 1, 2, 3, 4, 5, 6, 7, 8], 4)</code> should return <code>[[0, 1, 2, 3], [4, 5, 6, 7], [8]]</code>.');"
       ],
```
Comment (eltaxi) [inline seed/challenges/01-front-end-development-certification/basic-bonfires.json]: , 0D, 06, 22, 2D, 50, 44, 0C, 00    RF_CLIF_CFG_TECHNO_I_RX15693CLIF_SIGPRO_RM_CONFIG1_REG
#    A0, 0D, 04, 32, 03, 40, 3D            RF_CLIF_CFG_BR_106_I_TXA    CLIF_TRANSCEIVE_CONTROL_REG
#    A0, 0D, 06, 32, 42, F8, 10, FF, FF    RF_CLIF_CFG_BR_106_I_TXA    CLIF_ANA_TX_AMPLITUDE_REG
#    A0, 0D, 03, 32, 16, 00                RF_CLIF_CFG_BR_106_I_TXA    CLIF_TX_UNDERSHOOT_CONFIG_REG
#    A0, 0D, 03, 32, 15, 01                RF_CLIF_CFG_BR_106_I_TXA    CLIF_TX_OVERSHOOT_CONFIG_REG
#    A0, 0D, 03, 32, 0D, 22                RF_CLIF_CFG_BR_106_I_TXA    CLIF_TX_DATA_MOD_REG
#    A0, 0D, 03, 32, 14, 22                RF_CLIF_CFG_BR_106_I_TXA    CLIF_TX_SYMBOL23_MOD_REG
#    A0, 0D, 06, 32, 4A, 30, 07, 01, 1F    RF_CLIF_CFG_BR_106_I_TXA    CLIF_ANA_TX_SHAPE_CONTROL_REG
#    A0, 0D, 06, 34, 2D, 24, 77, 0C, 00    RF_CLIF_CFG_BR_106_I_RXA_P  CLIF_SIGPRO_RM_CONFIG1_REG
#    A0, 0D, 06, 34, 34, 00, 00, E4, 03    RF_CLIF_CFG_BR_106_I_RXA_P  CLIF_AGC_CONFIG1_REG
#    A0, 0D, 06, 34, 44, 21, 00, 02, 00    RF_CLIF_CFG_BR_106_I_RXA_P  CLIF_ANA_RX_REG
#    A0, 0D, 06, 35, 44, 21, 00, 02, 00    RF_CLIF_CFG_BR_106_I_RXA_P  CLIF_ANA_RX_REG
#    A0, 0D, 06, 38, 4A, 53, 07, 01, 1B    RF_CLIF_CFG_BR_212_I_TXA    CLIF_ANA_TX_SHAPE_CONTROL_REG
#    A0, 0D, 06, 38, 42, 68, 10, FF, FF    RF_CLIF_CFG_BR_212_I_TXA    CLIF_ANA_TX_AMPLITUDE_REG
#    A0, 0D, 03, 38, 16, 00                RF

**Comment by ltegman** [comment]
Please add commas after the new lines you added.

**Comment by ltegman** [comment]
Thanks for the correction! :smile: Would you mind [squashing your commits](https://gitter.im/FreeCodeCamp/Help) into one?

**Comment by ltegman** [comment]
Thanks! :+1:

**Pull request merged**